### PR TITLE
Added a script to get the details and filtered out the unwanted data 

### LINKED ISSUE #1: Python script to get the stats of an individual
Create a folder `scripts` in the repo and add the script which can get the details of the stats of a person.

**Help**
1. You can use [GitHub GraphQL API v4](https://developer.github.com/v4/) for fetching the data. 
2. Write a query that can get the required details, details of PRs made in October. Try using the [GraphQL API Explorer](https://developer.github.com/v4/explorer/).
3. Use Python to call the API request for fetching the raw data.
4. Parse the data and show the required results.

*Note*: We can integrate this script directly into the Flask App.

### PR DESCRIPTION
**Pull request by adiaux**
Added the query and accessed data of a GitHub user. Then filtered out the date and got the final data needed using regex  Fixes #1 . @vchrombie, if you have any ideas, please suggest

### CODE REVIEW

**Comment by vchrombie** [comment]
The PR looks fine but I need some changes. It would be great if you can complete them in this PR. Don't create a new PR. Let me know if you need any help. :slightly_smiling_face:

File: scripts/script.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import requests
+import json
+import collections
+import re 
+
+accessToken = "############################ "
+headers = {"Authorization": "bearer "+ accessToken }
+
+
+def getTopicList():
```
Comment (vchrombie) [inline scripts/script.py]: The function name should be something related instead of just a random name.

File: scripts/script.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        } 
+    }
+    } 
+    """
+    request = requests.post('https://api.github.com/graphql', json={'query': topic_query}, headers=headers)
+    if request.status_code == 200:
+        result = request.json()
+        for pull in result['data']['repositoryOwner']['pullRequests']['nodes']:
+            xx=pull['createdAt']
+            z = re.match(r'2019-09',xx)
```
Comment (vchrombie) [inline scripts/script.py]: Similarly, the variables' names should be meaningful.

File: scripts/script.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            if z:
+                print(pull['createdAt'])
+                print(pull['additions'])
+                print(pull['deletions'])
+            else:
+                print("No pull requests made for hacktober")
```
Comment (vchrombie) [inline scripts/script.py]: Maybe you can change the print statement to "No PRs made in Hacktoberfest"

File: scripts/script.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                break
+
+
+
+
```
Comment (vchrombie) [inline scripts/script.py]: remove all the extra spaces

**Comment by vchrombie** [comment]
Also, a few more suggestions. 
1. The commit message should tell about the change you have done in that particular commit.
2. The PR title should tell about the changes you have done in that PR in one line.
3. The PR description can be a detailed explanation of the PR. 

Your PR description looks fine bu the rest two are not. Please change your commit message and PR title.

**Comment by vchrombie** [comment]
Let me know if you need any help regarding anything. :slightly_smiling_face:

File: scripts/script.py (hunk 2 above)
Comment (vchrombie) [inline scripts/script.py]: Wow, `2019-09`. :open_mouth: 

09 - September!

File: scripts/script.py [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        for pull in result['data']['repositoryOwner']['pullRequests']['nodes']:
+            xx=pull['createdAt']
+            z = re.match(r'2019-09',xx)
+            if z:
+                print(pull['createdAt'])
+                print(pull['additions'])
+                print(pull['deletions'])
+            else:
+                print("No pull requests made for hacktober")
+                break
```
Comment (vchrombie) [inline scripts/script.py]: This logic doesn't seem to work. Check this again.

File: scripts/script.py [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    topic_query = """
+    query {
+    repositoryOwner(login:"vchrombie") {
+        login 
+        ... on User {
+        name
+        avatarUrl
+        pullRequests(last: 100){
+            nodes{
+            id
```
Comment (vchrombie) [inline scripts/script.py]: The query is requesting the `id` of the PR. Maybe you should make use out of it.

File: scripts/script.py (hunk 1 above)
Comment (SkaarFacee) [reply scripts/script.py]: Changed the function name, to make them more meaningful and user understandable.

File: scripts/script.py (hunk 2 above)
Comment (SkaarFacee) [reply scripts/script.py]: Variable names changed, for easier understanding

File: scripts/script.py (hunk 3 above)
Comment (SkaarFacee) [reply scripts/script.py]: Changed the print statement according to the need.

File: scripts/script.py (hunk 4 above)
Comment (SkaarFacee) [reply scripts/script.py]: All extra spaces are removed.

File: scripts/script.py (hunk 2 above)
Comment (SkaarFacee) [reply scripts/script.py]: Took care of that typo :astonished:

File: scripts/script.py (hunk 5 above)
Comment (SkaarFacee) [reply scripts/script.py]: Changed the logic of the code so that it can also handle alternative possibilities.

**Comment by vchrombie** [comment]
It looks impressive, after the changes. :smile: 

Final suggestion, change the PR title.

**Comment by adiaux** [comment]
Made all the needed changes. :bomb:

**Comment by vchrombie** [comment]
Nice work @adiaux. :slightly_smiling_face: 
LGTM. Merging it. :tada:

**Comment by vchrombie** [comment]
Also, one more suggestion. Whenever you are working on some issue and sending a PR, make sure you create a new branch `git checkout -b newbranch` and do your changes in that. Send a PR from that branch. 

It is advised not to use the master branch for experimenting.

**Comment by iammarco11** [comment]
@adiaux well done🤩

**Pull request merged**